.banner.txt

```diff
@@ -1,6 +1,7 @@
 /*!
  * <%= pkg.name %> library v<%= pkg.version %>
  * <%= pkg.homepage %>
+ * @preserve
```

Comment:
without this annotation `uglify-js` will strip the comment, see: https://github.com/mishoo/UglifyJS2#keeping-copyright-notices-or-other-comments